backend/src/main/java/WebApi.java

```diff
@@ -1,15 +1,79 @@
+import com.google.gson.Gson;
+import model.Customer;
+import model.Order;
+import model.OrderedPart;
+
 import static spark.Spark.*;
 
 /**
  * Starts a restapi att localhost:4567
  */
 public class WebApi {
-    private static WebInterface webInterface = new WebInterfaceImpl();
+    private static CustomerInterface ci = new CustomerController();
+    private static MaterialInterface mi;
+    private static PrintingInterface pi;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused "pi" private field. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1068)